MAGETWO-84124: Add bundle parent/child relationship during import

adam-paterson opened the pull request:
<!--- Provide a general summary of the Pull Request in the Title above -->

### Description
* Save parent/child relationships for bundled products during import process.

### Fixed Issues (if relevant)
<!--- Provide a list of fixed issues in the format magento/magento2#<issue_number>, if relevant  -->
1. magento/magento2#12330: Imported bundle products are not assigned stock status
2. #80 

### Manual testing scenarios
<!--- Provide a set of unambiguous steps to test the proposed code change -->
1. Create Bundled product with options
2. Export product via System -> Import/Export
3. Re-upload CSV choosing "Replace" method

### Contribution checklist
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

adam-paterson committed:
Allow tablerates csv to be read from non default tmp directory

slavvka committed:
DEVOPS-2174: Fix integration tests

slavvka committed:
DEVOPS-2174: Fix integration tests

rogyar committed:
Fix aggregations use statements and return values

Commit:
Fixed code style issues

magento-engcom-team committed:
ENGCOM-1408: [Backport] Fix aggregations use statements and return values #14894

 - Merge Pull Request magento/magento2#14894 from rogyar/magento2:2.1-aggregations-fix
 - Merged commits:
   1. 9b1beab5bd435da62d97ea27a4522b405573a980
   2. 912c06cfe0b03e7efb2b29432bd6a30980ec60f1

Commit:
ENGCOM-1408: [Backport] Fix aggregations use statements and return values #14894

magento-engcom-team committed:
:arrows_clockwise: [EngCom] Public Pull Requests - 2.1-develop

Accepted Public Pull Requests:
 - magento/magento2#14894: [Backport] Fix aggregations use statements and return values (by @rogyar)

rogyar committed:
Disable add to cart button when redirect to cart enabled

rogyar committed:
Temporary variable removed

rogyar committed:
Fixed issue: prices aren't readable when using custom price symbol

magento-engcom-team committed:
ENGCOM-1410: [Backport] Prices aren't readable when using custom price symbol #14902

 - Merge Pull Request magento/magento2#14902 from rogyar/magento2:2.1-admin-custom-currency-fix
 - Merged commits:
   1. 5c5859566074a7e0be2fb7fbb7b326cd0f90d937

magento-engcom-team committed:
ENGCOM-1411: [Backport] Refactoring: remove unuseful temporary variable #14901

 - Merge Pull Request magento/magento2#14901 from rogyar/magento2:2.1-categorysave-unnecessary-var
 - Merged commits:
   1. b3a7d7f1b52cf880f0c82eafb5a3b64b6c88fdea

magento-engcom-team committed:
ENGCOM-1412: [Backport] Disable add to cart button when redirect to cart enabled #14899

 - Merge Pull Request magento/magento2#14899 from rogyar/magento2:2.1-addtocart-redirect-fix
 - Merged commits:
   1. 3cfe20df6c5f756f030c018a99afea1d480be750

rogyar committed:
Pass parameter for export button url

Commit:
ENGCOM-1411: [Backport] Refactoring: remove unuseful temporary variable #14901

rogyar committed:
Switch updatecart qty input validators to dynamic instead of hardcoded value

magento-engcom-team committed:
ENGCOM-1421: [Backport] Switch updatecart qty input validators to dynamic #14911

 - Merge Pull Request magento/magento2#14911 from rogyar/magento2:2.1-improve-cart-qty-validation
 - Merged commits:
   1. 838f08fee38592e3604ecbe01de7a4531d7321ed

rogyar committed:
Changed method scope

magento-engcom-team committed:
:arrows_clockwise: [EngCom] Public Pull Requests - 2.1-develop

Accepted Public Pull Requests:
 - magento/magento2#14901: [Backport] Refactoring: remove unuseful temporary variable (by @rogyar)

magento-team committed:
MAGETWO-88697: Magento 2.1.13 Publication

 (build 2.1.13.013)

rogyar committed:
Active argument removed

magento-engcom-team committed:
ENGCOM-1413: [Backport] Pass parameter for export button url #14903

 - Merge Pull Request magento/magento2#14903 from rogyar/magento2:2.1-order-grid-export-fix
 - Merged commits:
   1. 4bb9d270b7d8b383a69f2e2101fd0111c24b19f7
   2. 67268ca4395a577a3d7d1ae17fdc455691fef325
   3. 9c5517aa6ba95dc8070555a390105faf41c404c7

Commit:
ENGCOM-1421: [Backport] Switch updatecart qty input validators to dynamic #14911

Commit:
ENGCOM-1412: [Backport] Disable add to cart button when redirect to cart enabled #14899

magento-engcom-team committed:
:arrows_clockwise: [EngCom] Public Pull Requests - 2.1-develop

Accepted Public Pull Requests:
 - magento/magento2#14911: [Backport] Switch updatecart qty input validators to dynamic (by @rogyar)
 - magento/magento2#14899: [Backport] Disable add to cart button when redirect to cart enabled (by @rogyar)

Commit:
ENGCOM-1410: [Backport] Prices aren't readable when using custom price symbol #14902

magento-engcom-team committed:
:arrows_clockwise: [EngCom] Public Pull Requests - 2.1-develop

Accepted Public Pull Requests:
 - magento/magento2#14902: [Backport] Prices aren't readable when using custom price symbol (by @rogyar)


Fixed GitHub Issues:
 - magento/magento2#12430: While assigning prices to configurable products, prices aren's readable when using custom price symbol. (reported by @quisse) has been fixed in magento/magento2#14902 by @rogyar in 2.1-develop branch
   Related commits:
     1. 5c5859566074a7e0be2fb7fbb7b326cd0f90d937

rogyar committed:
Observer name typo fix

magento-engcom-team committed:
ENGCOM-1428: [Backport] Customer observer name typo fix #14922

 - Merge Pull Request magento/magento2#14922 from rogyar/magento2:2.1-observer-name-typo
 - Merged commits:
   1. db28e02f088393d328f0d8d4ccf39fe9ed10ab74

bydrei committed:
ISSUE-14663 : Preserve user group id when using /V1/customers/:customerId

magento-engcom-team committed:
ENGCOM-1337: Preserve user group id when using /V1/customers/:customerId (PUT) #14757

 - Merge Pull Request magento/magento2#14757 from ferrazzuk/magento2:Issue-14663
 - Merged commits:
   1. 51f2f076a8fddbeb7b29914a1994236c29bdc369

rogyar committed:
Fix faulty admin spinner animation

magento-engcom-team committed:
ENGCOM-1433: [Backport] Fix faulty admin spinner animation #14940

 - Merge Pull Request magento/magento2#14940 from rogyar/magento2:2.1-spinner-animation-fix
 - Merged commits:
   1. 91857b0c6537c17d3e97fbc6c84e25c41e9b1feb

VladimirZaets committed:
ENGCOM-1428: [Backport] Customer observer name typo fix #14922

VladimirZaets committed:
ENGCOM-1413: [Backport] Pass parameter for export button url #14903

magento-engcom-team committed:
:arrows_clockwise: [EngCom] Public Pull Requests - 2.1-develop

Accepted Public Pull Requests:
 - magento/magento2#14922: [Backport] Customer observer name typo fix (by @rogyar)
 - magento/magento2#14903: [Backport] Pass parameter for export button url (by @rogyar)


Fixed GitHub Issues:
 - magento/magento2#12714: Extra records are in exported CSV file for order (reported by @alena-marchenko) has been fixed in magento/magento2#14903 by @rogyar in 2.1-develop branch
   Related commits:
     1. 4bb9d270b7d8b383a69f2e2101fd0111c24b19f7
     2. 67268ca4395a577a3d7d1ae17fdc455691fef325
     3. 9c5517aa6ba95dc8070555a390105faf41c404c7

Commit:
ENGCOM-1433: [Backport] Fix faulty admin spinner animation #14940

magento-engcom-team committed:
:arrows_clockwise: [EngCom] Public Pull Requests - 2.1-develop

Accepted Public Pull Requests:
 - magento/magento2#14940: [Backport] Fix faulty admin spinner animation (by @rogyar)

omiroshnichenko committed:
MAGETWO-88714: Merge release branch into 2.1-develop
Merge branch '2.1' of github.com:magento/magento2ce into 2.1-develop-update

okolesnyk committed:
Merge pull request #2487 from magento-plankton/2.1-develop-update

[Plankton] Merge release branch to 2.1-develop

dverkade committed:
Fix for displaying a negative price for a custom option. Currently a negative price is displayed as +- 5.00 for instance. By changing the template to check of the value is actually positive a negative value will be displayed as - 5.00 and a positive value will be displayed as + 5.00

slavvka committed:
Merge remote-tracking branch 'mainline/2.1-develop' into DEVOPS-2174-2.1

VladimirZaets committed:
ENGCOM-1337: Preserve user group id when using /V1/customers/:customerId (PUT) #14757

slavvka committed:
DEVOPS-2174: Fix integration tests

magento-engcom-team committed:
:arrows_clockwise: [EngCom] Public Pull Requests - 2.1-develop

Accepted Public Pull Requests:
 - magento/magento2#14757: Preserve user group id when using /V1/customers/:customerId (PUT) (by @ferrazzuk)


Fixed GitHub Issues:
 - magento/magento2#14663: Updating Customer through rest/all/V1/customers/:id resets group_id if group_id not passed in payload (reported by @abancroft0) has been fixed in magento/magento2#14757 by @ferrazzuk in 2.1-develop branch
   Related commits:
     1. 51f2f076a8fddbeb7b29914a1994236c29bdc369

Commit:
merge magento/2.1-develop into magento-borg/DEVOPS-2174-2.1

slavvka committed:
Merge pull request #2497 from magento-borg/DEVOPS-2174-2.1

Fixed issues:
- DEVOPS-232 Static Testing in Jenkins

rogyar committed:
Minicart should require dropdownDialog

magento-engcom-team committed:
ENGCOM-1459: [Backport] Minicart should require dropdownDialog #15025

 - Merge Pull Request magento/magento2#15025 from rogyar/magento2:2.1-require-dropdown-minicart
 - Merged commits:
   1. 938091d56ec15e7bcd0a1a2ecd85b5c0bc10968d

Commit:
ENGCOM-1459: [Backport] Minicart should require dropdownDialog #15025

magento-engcom-team committed:
:arrows_clockwise: [EngCom] Public Pull Requests - 2.1-develop

Accepted Public Pull Requests:
 - magento/magento2#15025: [Backport] Minicart should require dropdownDialog (by @rogyar)

Commit:
Correct missing commits for #10210

Add missing commits for \app\code\Magento\Sales\Model\Order\Email\Sender\OrderSender.php

magento-engcom-team committed:
ENGCOM-1465: [2.1-develop][Backport] Transport variable can not be altered in email_invoice_set_template_vars_before Event #15038

 - Merge Pull Request magento/magento2#15038 from gwharton/magento2:2.1-develop-10210
 - Merged commits:
   1. 9a95dae80232792eb5298b240f86b342c70a7341

IgorVitol committed:
Add 'const' type support to layout arguments

rogyar committed:
Add statement to 'beforeSave' method to allow app:config:import

Commit:
Don't recalculate tax for every price on category page

magento-engcom-team committed:
ENGCOM-1475: [Backport] Add statement to 'beforeSave' method to allow app:config:import #15066

 - Merge Pull Request magento/magento2#15066 from rogyar/magento2:2.1-config-import-country-card
 - Merged commits:
   1. a7e8b71095890e0f0ebd1ac400c76f5d4f9aaa5c

rogyar committed:
JSCS fix: empty line above ‘return’ is required

magento-engcom-team committed:
ENGCOM-1491: [Backport] Fix infinite checkout loader on a script error #15091

 - Merge Pull Request magento/magento2#15091 from rogyar/magento2:2.1-checkout-loader-fix
 - Merged commits:
   1. 78a2e51ab57f399fdc2076d5b9017420255243d9

rogyar committed:
Changed return type of addToCartPostParams to array

rogyar committed:
Duplicate Order Confirmation Emails for PayPal Express checkout order

magento-engcom-team committed:
ENGCOM-1492: [Backport] Duplicate Order Confirmation Emails for PayPal Express checkout order #15094

 - Merge Pull Request magento/magento2#15094 from rogyar/magento2:2.1-paypal-duplicate-email-fix
 - Merged commits:
   1. 0799789a0f53e055ed02d2ea3afc9745b00ebf4e

Commit:
Removed extra spaces from language file

magento-engcom-team committed:
ENGCOM-1494: [Backport] Removed extra spaces from language file #15099

 - Merge Pull Request magento/magento2#15099 from sergiy-v/magento2:feature/catalog-inventory-language-file-fix
 - Merged commits:
   1. 21bd884d540d77a1d22eee9f38de684ab31b9bc6

Commit:
stripped filter value

magento-engcom-team committed:
ENGCOM-1496: [Backport] Changed return type of addToCartPostParams to array #15093

 - Merge Pull Request magento/magento2#15093 from rogyar/magento2:2.1-product-list-return-hint-fix
 - Merged commits:
   1. 7104b91ddfa33de303362322a551ca1553feb902

magento-engcom-team committed:
ENGCOM-1497: [Backport] FIX for issue#14855 - Adding an * to do a customer search #15101

 - Merge Pull Request magento/magento2#15101 from sergiy-v/magento2:feature/2.1-asterisk-on-grid-search
 - Merged commits:
   1. ceb486d1adae27befc29f22ce8188de1f1a32e71

Commit:
move customer.account.dashboard.info.extra block to contact information

Commit:
Removed extra close tag

Commit:
Change 'Update'-button visibility on change qty event

Commit:
Renamed config setting to represent what it actually does

magento-engcom-team committed:
ENGCOM-1498: [Backport] Removed extra close tag #15103

 - Merge Pull Request magento/magento2#15103 from sergiy-v/magento2:feature/2.1-magento-review-view-template-remove-extra-tag
 - Merged commits:
   1. 8823f96b0589029c0ac495a34dcab6721fcf0b3c

Commit:
changed path for templates

Commit:
Fix typo in less button definition

Commit:
fixed typos in less files

magento-engcom-team committed:
ENGCOM-1499: [Backport] Change 'Update'-button visibility on change qty event #15104

 - Merge Pull Request magento/magento2#15104 from sergiy-v/magento2:feature/2.1-minicart-update-button
 - Merged commits:
   1. 45b8f87cce6751551c98f11b1562873b4512ca3a

magento-engcom-team committed:
ENGCOM-1500: [Backport] use 'Module_Name::template/path' format instead of using template/path #15106

 - Merge Pull Request magento/magento2#15106 from sergiy-v/magento2:feature/2.1-magento-review-renderer-path
 - Merged commits:
   1. 20993c1af8dfdf8ef06699150067469fcaa6893c

magento-engcom-team committed:
ENGCOM-1502: [Backport] Fix typo in less button definition  #15108

 - Merge Pull Request magento/magento2#15108 from sergiy-v/magento2:feature/2.1-buttons-less-typo
 - Merged commits:
   1. 12bf629f5faa4110e2f7a0c87e9bc98f2487d464

magento-engcom-team committed:
ENGCOM-1503: [Backport] Fixed typos in .less files #15109

 - Merge Pull Request magento/magento2#15109 from sergiy-v/magento2:feature/2.1-port-15023
 - Merged commits:
   1. e2f70fecfe8bd013b89c5cb52abce310659b15b6

magento-engcom-team committed:
ENGCOM-1504: [Backport] 'Add Block Names to Hints' config setting to represent what it actually does #15105

 - Merge Pull Request magento/magento2#15105 from sergiy-v/magento2:feature/2.1-block-class-type-config-label
 - Merged commits:
   1. 841b3225f8c4f192e58f0eba1aaf518d0cb91c31

Commit:
ENGCOM-1504: [Backport] "Add Block Names to Hints" config setting to represent what it actually does #15105

Commit:
ENGCOM-1503: [Backport] Fixed typos in .less files #15109

Commit:
ENGCOM-1502: [Backport] Fix typo in less button definition  #15108

Commit:
ENGCOM-1496: [Backport] Changed return type of addToCartPostParams to array #15093

Commit:
ENGCOM-1494: [Backport] Removed extra spaces from language file #15099

magento-engcom-team committed:
ENGCOM-1506: [Backport] Move customer.account.dashboard.info.extra block to contact information #15102

 - Merge Pull Request magento/magento2#15102 from sergiy-v/magento2:feature/2.1-move-account-dashboard-extra-info-to-contact-block
 - Merged commits:
   1. 9a1d9ded445b9b555d213a63f69700328a0893dd

magento-engcom-team committed:
:arrows_clockwise: [EngCom] Public Pull Requests - 2.1-develop

Accepted Public Pull Requests:
 - magento/magento2#15105: [Backport] "Add Block Names to Hints" config setting to represent what it actually does (by @sergiy-v)
 - magento/magento2#15109: [Backport] Fixed typos in .less files (by @sergiy-v)
 - magento/magento2#15108: [Backport] Fix typo in less button definition  (by @sergiy-v)
 - magento/magento2#15093: [Backport] Changed return type of addToCartPostParams to array (by @rogyar)
 - magento/magento2#15099: [Backport] Removed extra spaces from language file (by @sergiy-v)

Commit:
ENGCOM-1506: [Backport] Move customer.account.dashboard.info.extra block to contact information #15102

Commit:
ENGCOM-1500: [Backport] use "Module_Name::template/path" format instead of using template/path #15106

Commit:
ENGCOM-1475: [Backport] Add statement to 'beforeSave' method to allow app:config:import #15066

magento-engcom-team committed:
:arrows_clockwise: [EngCom] Public Pull Requests - 2.1-develop

Accepted Public Pull Requests:
 - magento/magento2#15102: [Backport] Move customer.account.dashboard.info.extra block to contact information (by @sergiy-v)
 - magento/magento2#15106: [Backport] use "Module_Name::template/path" format instead of using template/path (by @sergiy-v)
 - magento/magento2#15066: [Backport] Add statement to 'beforeSave' method to allow app:config:import (by @rogyar)

rogyar committed:
Add concrete type hints for product and category resources

dmytro-ch committed:
[Backport] Fix typo in input type variable name

magento-engcom-team committed:
ENGCOM-1522: [Backport] Add concrete type hints for product and category resources #15138

 - Merge Pull Request magento/magento2#15138 from rogyar/magento2:2.1-catalog-type-hints
 - Merged commits:
   1. a1f1a0fc38edfe5ba134b64bd26ca2cf50cf91d1

magento-engcom-team committed:
ENGCOM-1523: [Backport] Fix typo in input type variable name #15146

 - Merge Pull Request magento/magento2#15146 from dmytro-ch/magento2:fix/2.1-typo-in-input-type-variable-name
 - Merged commits:
   1. 28d4897457d8f82ffa83303d8e960bc58bf0fe43

VladimirZaets committed:
ENGCOM-1523: [Backport] Fix typo in input type variable name #15146

VladimirZaets committed:
ENGCOM-1522: [Backport] Add concrete type hints for product and category resources #15138

magento-engcom-team committed:
:arrows_clockwise: [EngCom] Public Pull Requests - 2.1-develop

Accepted Public Pull Requests:
 - magento/magento2#15146: [Backport] Fix typo in input type variable name (by @dmytro-ch)
 - magento/magento2#15138: [Backport] Add concrete type hints for product and category resources (by @rogyar)

rogyar committed:
Fix typo in doc for updateSpecificCoupons

magento-engcom-team committed:
ENGCOM-1525: [Backport] Fix typo in doc for updateSpecificCoupons #15157

 - Merge Pull Request magento/magento2#15157 from rogyar/magento2:2.1-doc-typo-coupon
 - Merged commits:
   1. d2a4c73c358d15804ae4c0cf7bb2ce3a369fd8a7

rogyar committed:
Default schedule config for sitemap_generate job added

magento-engcom-team committed:
ENGCOM-1528: [Backport] Default schedule config for sitemap_generate job added #15159

 - Merge Pull Request magento/magento2#15159 from rogyar/magento2:2.1-default-schedule-sitemap
 - Merged commits:
   1. f4ae0e32abc1de2cfc5db23260df0e82ee68f75a

Commit:
ENGCOM-1499: [Backport] Change 'Update'-button visibility on change qty event #15104

Commit:
ENGCOM-1497: [Backport] FIX for issue#14855 - Adding an * to do a customer search #15101

Commit:
ENGCOM-1491: [Backport] Fix infinite checkout loader on a script error #15091

KaushikChavda committed:
Swatches module file was missing in 2.1

magento-engcom-team committed:
ENGCOM-1533: Swatches module file was missing in 2.1 #15169

 - Merge Pull Request magento/magento2#15169 from kaushik-chavda/magento2:swatches
 - Merged commits:
   1. 4d4630db03dcdde32177354ce45edb503281dc7b

Commit:
magento/magento2#15101

Commit:
magento/magento2#15101

hostep committed:
Fixed format of purchase date in order grid again.

magento-engcom-team committed:
:arrows_clockwise: [EngCom] Public Pull Requests - 2.1-develop

Accepted Public Pull Requests:
 - magento/magento2#15104: [Backport] Change 'Update'-button visibility on change qty event (by @sergiy-v)
 - magento/magento2#15101: [Backport] FIX for issue#14855 - Adding an * to do a customer search (by @sergiy-v)
 - magento/magento2#15091: [Backport] Fix infinite checkout loader on a script error (by @rogyar)

magento-engcom-team committed:
ENGCOM-1536: Fixed format of purchase date in order grid again. #15176

 - Merge Pull Request magento/magento2#15176 from hostep/magento2:fixed-pr-10260-after-2.1.8
 - Merged commits:
   1. c5bfca6ecf3cfce560f271d335ead6ca0929d3b0

rogyar committed:
Corrected params in comment block

dmytro-ch committed:
[Backport] Removed unused class declaration and code

magento-engcom-team committed:
ENGCOM-1544: [Backport] Corrected param in docblock #15183

 - Merge Pull Request magento/magento2#15183 from rogyar/magento2:2.1-urlinterface-docblock-fix
 - Merged commits:
   1. d6248a2568d296fb24dcbcc906c83db42255ce16

magento-engcom-team committed:
ENGCOM-1546: [Backport] Removed unused class declaration and code #15184

 - Merge Pull Request magento/magento2#15184 from dmytro-ch/magento2:fix/2.1-unused-class-declaration-and-code
 - Merged commits:
   1. 236e1a746fcb19407d78ea9550d8e1ff004f6a8f

VladimirZaets committed:
ENGCOM-1546: [Backport] Removed unused class declaration and code #15184

VladimirZaets committed:
ENGCOM-1544: [Backport] Corrected param in docblock #15183

VladimirZaets committed:
ENGCOM-1533: Swatches module file was missing in 2.1 #15169

VladimirZaets committed:
ENGCOM-1498: [Backport] Removed extra close tag #15103

VladimirZaets committed:
ENGCOM-1492: [Backport] Duplicate Order Confirmation Emails for PayPal Express checkout order #15094

magento-engcom-team committed:
ENGCOM-1554: [Backport 2.1] Fix for displaying a negative price for a custom option #15203

 - Merge Pull Request magento/magento2#15203 from dverkade/magento2:2.1-Fix-negative-value-for-custom-option
 - Merged commits:
   1. c224ad068186e3398ec7f048352f19f648fdbb58

VladimirZaets committed:
:arrows_clockwise: [EngCom] Public Pull Requests - 2.1-develop

Accepted Public Pull Requests:
 - magento/magento2#15184: [Backport] Removed unused class declaration and code (by @dmytro-ch)
 - magento/magento2#15183: [Backport] Corrected param in docblock (by @rogyar)
 - magento/magento2#15169: Swatches module file was missing in 2.1 (by @kaushik-chavda)
 - magento/magento2#15103: [Backport] Removed extra close tag (by @sergiy-v)
 - magento/magento2#15094: [Backport] Duplicate Order Confirmation Emails for PayPal Express checkout order (by @rogyar)

VladimirZaets committed:
ENGCOM-1525: [Backport] Fix typo in doc for updateSpecificCoupons #15157

magento-engcom-team committed:
:arrows_clockwise: [EngCom] Public Pull Requests - 2.1-develop

Accepted Public Pull Requests:
 - magento/magento2#15157: [Backport] Fix typo in doc for updateSpecificCoupons (by @rogyar)

rogyar committed:
Translate action Label

rogyar committed:
Fixed datepicker problem when using non en-US locale.

magento-engcom-team committed:
ENGCOM-1564: [Backport] Translate action Label #15221

 - Merge Pull Request magento/magento2#15221 from rogyar/magento2:2.1-translate-action-label
 - Merged commits:
   1. 07dae799939753fdceef2e9c0c90565b13a16b53

magento-engcom-team committed:
ENGCOM-1565: [Backport] Fixed datepicker problem when using non en-US locale. #15222

 - Merge Pull Request magento/magento2#15222 from rogyar/magento2:2.1-non-us-locale-datepicker
 - Merged commits:
   1. b75179297c8a47d882500734bef231c8a10835b3

VitaliyBoyko committed:
Fix typo in design rule hint message

VitaliyBoyko committed:
Add missing translations in Magento_UI

VitaliyBoyko committed:
Fixed double space typo

VitaliyBoyko committed:
Changed constructor typo in Javascript class

VitaliyBoyko committed:
Fixed php notice when invalid ui_component config is used

Commit:
Remove unused namespace from ui export

magento-engcom-team committed:
ENGCOM-1566: Remove unused namespace from ui export #15240

 - Merge Pull Request magento/magento2#15240 from Atwix/magento2:remove-unused-namespace-from-ui-export
 - Merged commits:
   1. 3d3f9227af16373e08775108d5e6c5c3e7204582

magento-engcom-team committed:
ENGCOM-1567: [Backport] Fix typo in design rule hint message #15233

 - Merge Pull Request magento/magento2#15233 from VitaliyBoyko/magento2:fix/design-rule-typo
 - Merged commits:
   1. 999441026773178bb6b8b7aa422c5828cc78164c

magento-engcom-team committed:
ENGCOM-1568: [Backport] Add missing translations in Magento_UI #15234

 - Merge Pull Request magento/magento2#15234 from VitaliyBoyko/magento2:11440-Add-missing-translations-in-Magento_UI
 - Merged commits:
   1. 0c2e4961bb9f8532e400720370a3cfdce982fe99

magento-engcom-team committed:
ENGCOM-1569: [Backport] Fixed double space typo #15235

 - Merge Pull Request magento/magento2#15235 from VitaliyBoyko/magento2:11569-Fixed_double_space_typo
 - Merged commits:
   1. d41e2c535be3805b9fbc209dfde3459cd7ce3b0b

magento-engcom-team committed:
ENGCOM-1570: [Backport] Fixed php notice when invalid ui_component config is used #15238

 - Merge Pull Request magento/magento2#15238 from VitaliyBoyko/magento2:Fixed-php-notice-when-invalid-ui_component-config-is-used
 - Merged commits:
   1. 59806705d83290d119de4cf77c0a6fd833156722

magento-engcom-team committed:
ENGCOM-1571: [Backport] Changed constructor typo in Javascript class #15237

 - Merge Pull Request magento/magento2#15237 from VitaliyBoyko/magento2:Changed-constructor-typo-in-Javascript-class
 - Merged commits:
   1. 76c44636eb03efe7b9230cebdc9a86c0d23b327d

Commit:
ENGCOM-1566: Remove unused namespace from ui export #15240

Commit:
ENGCOM-1554: [Backport 2.1] Fix for displaying a negative price for a custom option #15203

Commit:
ENGCOM-1536: Fixed format of purchase date in order grid again. #15176

magento-engcom-team committed:
:arrows_clockwise: [EngCom] Public Pull Requests - 2.1-develop

Accepted Public Pull Requests:
 - magento/magento2#15240: Remove unused namespace from ui export (by @yuriyDne)
 - magento/magento2#15203: [Backport 2.1] Fix for displaying a negative price for a custom option (by @dverkade)
 - magento/magento2#15176: Fixed format of purchase date in order grid again. (by @hostep)

Commit:
ENGCOM-1565: [Backport] Fixed datepicker problem when using non en-US locale. #15222

Commit:
ENGCOM-1564: [Backport] Translate action Label #15221

magento-engcom-team committed:
:arrows_clockwise: [EngCom] Public Pull Requests - 2.1-develop

Accepted Public Pull Requests:
 - magento/magento2#15222: [Backport] Fixed datepicker problem when using non en-US locale. (by @rogyar)
 - magento/magento2#15221: [Backport] Translate action Label (by @rogyar)

VladimirZaets committed:
ENGCOM-1528: [Backport] Default schedule config for sitemap_generate job added #15159

magento-engcom-team committed:
:arrows_clockwise: [EngCom] Public Pull Requests - 2.1-develop

Accepted Public Pull Requests:
 - magento/magento2#15159: [Backport] Default schedule config for sitemap_generate job added (by @rogyar)


Fixed GitHub Issues:
 - magento/magento2#5768: Magento 2.0.7 XML sitemap is not generated by schedule (reported by @dexterityzx) has been fixed in magento/magento2#15159 by @rogyar in 2.1-develop branch
   Related commits:
     1. f4ae0e32abc1de2cfc5db23260df0e82ee68f75a

VladimirZaets committed:
ENGCOM-1571: [Backport] Changed constructor typo in Javascript class #15237

VladimirZaets committed:
ENGCOM-1569: [Backport] Fixed double space typo #15235

magento-engcom-team committed:
:arrows_clockwise: [EngCom] Public Pull Requests - 2.1-develop

Accepted Public Pull Requests:
 - magento/magento2#15237: [Backport] Changed constructor typo in Javascript class (by @VitaliyBoyko)
 - magento/magento2#15235: [Backport] Fixed double space typo (by @VitaliyBoyko)

dmytro-ch committed:
[Backport] Fix typos in variable names

dmytro-ch committed:
[Backport] Fix typos in PHPDocs and comments

Commit:
ENGCOM-1570: [Backport] Fixed php notice when invalid ui_component config is used #15238

Commit:
ENGCOM-1568: [Backport] Add missing translations in Magento_UI #15234

Commit:
ENGCOM-1567: [Backport] Fix typo in design rule hint message #15233

VladimirZaets committed:
Fix copyright

sanganinamrata committed:
Fixed typo mistake in function comment

magento-engcom-team committed:
:arrows_clockwise: [EngCom] Public Pull Requests - 2.1-develop

Accepted Public Pull Requests:
 - magento/magento2#15238: [Backport] Fixed php notice when invalid ui_component config is used (by @VitaliyBoyko)
 - magento/magento2#15234: [Backport] Add missing translations in Magento_UI (by @VitaliyBoyko)
 - magento/magento2#15233: [Backport] Fix typo in design rule hint message (by @VitaliyBoyko)

sanganinamrata committed:
Removed extra added space in function comment and fixed typo mistakes.

sanganinamrata committed:
Revert back file changes

sanganinamrata committed:
Revert back file changes.

rogyar committed:
Eliminate usage of "else" statements. Code improvements

rogyar committed:
Allow to configure min and max dates for date picker component

VladimirZaets committed:
ENGCOM-1465: [2.1-develop][Backport] Transport variable can not be altered in email_invoice_set_template_vars_before Event #15038

rogyar committed:
Fixed readme minor typo

magento-engcom-team committed:
:arrows_clockwise: [EngCom] Public Pull Requests - 2.1-develop

Accepted Public Pull Requests:
 - magento/magento2#15038: [2.1-develop][Backport] Transport variable can not be altered in email_invoice_set_template_vars_before Event (by @gwharton)


Fixed GitHub Issues:
 - magento/magento2#10210: Transport variable can not be altered in email_invoice_set_template_vars_before Event (reported by @diybook) has been fixed in magento/magento2#15038 by @gwharton in 2.1-develop branch
   Related commits:
     1. 9a95dae80232792eb5298b240f86b342c70a7341

slavvka committed:
MAGETWO-91894: Backport Static Test Fixes in CE

slavvka committed:
Merge pull request #2562 from magento-borg/MAGETWO-91894-2.1

Fixed issues:
- MAGETWO-91894 Backport Static Test Fixes in CE

yogeshsuhagiya committed:
Added language translation for message string

yogeshsuhagiya committed:
Format the code

VitaliyBoyko committed:
Alignment Array assignment

VitaliyBoyko committed:
Fix typo in securityCheckers array

VitaliyBoyko committed:
Typo in SSL port number

VitaliyBoyko committed:
Fix incorrect phpdoc return type

magento-engcom-team committed:
ENGCOM-1590: [Backport] Added hyphenation cutting edge to cutting-edge #15324

 - Merge Pull Request magento/magento2#15324 from rogyar/magento2:2.1-readme-typo-cuttingedge
 - Merged commits:
   1. 1b2d726c6bc007e3d41852d891b5ca8a72a83c58

VitaliyBoyko committed:
Unused variable removed

yogeshsuhagiya committed:
Added language translation key

yogeshsuhagiya committed:
Added language translation key

magento-engcom-team committed:
ENGCOM-1594: [Backport] Unused variable removed #15385

 - Merge Pull Request magento/magento2#15385 from VitaliyBoyko/magento2:Unused-variable-removed
 - Merged commits:
   1. 2a547a2381b0253ff513bfff2581c212841b8965

magento-engcom-team committed:
ENGCOM-1595: [Backport] Fix incorrect phpdoc return type #15380

 - Merge Pull Request magento/magento2#15380 from VitaliyBoyko/magento2:fix-incorrect-phpdoc-return-type
 - Merged commits:
   1. d8fd4ee6797a22e6ddb6c9f7183786da89e0d933

magento-engcom-team committed:
ENGCOM-1596: [Backport] Typo in SSL port number #15377

 - Merge Pull Request magento/magento2#15377 from VitaliyBoyko/magento2:Typo-in-SSL-port-number
 - Merged commits:
   1. e74bea128e7dcd48945dba24568f705050de4c8c

magento-engcom-team committed:
ENGCOM-1597: [Backport] Fix typo in securityCheckers array #15375

 - Merge Pull Request magento/magento2#15375 from VitaliyBoyko/magento2:Fix-typo-in-securityCheckers-array
 - Merged commits:
   1. 27e74b97261ec4c6e6a494b3d30d7b46ce6cfec8

magento-engcom-team committed:
ENGCOM-1598: [Backport] Alignment Array assignment #15363

 - Merge Pull Request magento/magento2#15363 from VitaliyBoyko/magento2:Alignement_Array_assignement
 - Merged commits:
   1. 11c90810c625dff047cddd2f7f57cb8af8fc5eed

magento-engcom-team committed:
ENGCOM-1600: [Backport] Fix typos in PHPDocs and comments #15299

 - Merge Pull Request magento/magento2#15299 from dmytro-ch/magento2:fix/2.1-typos-in-phpdocs-and-comments
 - Merged commits:
   1. ee90e5406a6a15bb71c142cf3cb89ac0da5cdf01

magento-engcom-team committed:
ENGCOM-1601: [Backport] Fix typos in variable names #15298

 - Merge Pull Request magento/magento2#15298 from dmytro-ch/magento2:fix/2.1-typos-in-variable-names
 - Merged commits:
   1. a5e014a5a5576dabb042acbf15410844993800e7

Commit:
ENGCOM-1601: [Backport] Fix typos in variable names #15298

Commit:
ENGCOM-1600: [Backport] Fix typos in PHPDocs and comments #15299

Commit:
ENGCOM-1598: [Backport] Alignment Array assignment #15363

Commit:
ENGCOM-1597: [Backport] Fix typo in securityCheckers array #15375

Commit:
ENGCOM-1596: [Backport] Typo in SSL port number #15377

Commit:
ENGCOM-1595: [Backport] Fix incorrect phpdoc return type #15380

Commit:
ENGCOM-1594: [Backport] Unused variable removed #15385

Commit:
ENGCOM-1590: [Backport] Added hyphenation cutting edge to cutting-edge #15324

Commit:
Fix typos in PHPDocs and comments

Commit:
:arrows_clockwise: [EngCom] Public Pull Requests - 2.1-develop

Accepted Public Pull Requests:
 - magento/magento2#15298: [Backport] Fix typos in variable names (by @dmytro-ch)
 - magento/magento2#15299: [Backport] Fix typos in PHPDocs and comments (by @dmytro-ch)
 - magento/magento2#15363: [Backport] Alignment Array assignment (by @VitaliyBoyko)
 - magento/magento2#15375: [Backport] Fix typo in securityCheckers array (by @VitaliyBoyko)
 - magento/magento2#15377: [Backport] Typo in SSL port number (by @VitaliyBoyko)
 - magento/magento2#15380: [Backport] Fix incorrect phpdoc return type (by @VitaliyBoyko)
 - magento/magento2#15385: [Backport] Unused variable removed (by @VitaliyBoyko)
 - magento/magento2#15324: [Backport] Added hyphenation cutting edge to cutting-edge (by @rogyar)

dmytro-ch committed:
[Backport] Remove non-existing argument

magento-engcom-team committed:
ENGCOM-1612: [Backport] Remove non-existing argument #15394

 - Merge Pull Request magento/magento2#15394 from dmytro-ch/magento2:fix/2.1-remove-non-existing-argument
 - Merged commits:
   1. 3c89d4541f9cb1ead751dc8289c16492f0c0e4f0

magento-engcom-team committed:
ENGCOM-1614: [Backport] Fixed typo mistake in function comment #15396

 - Merge Pull Request magento/magento2#15396 from dmytro-ch/magento2:2.1-develop-PR-port-15302
 - Merged commits:
   1. 2c31e4d89ca1e3af89e224cbd355f50861d7b615
   2. 32c4a337aa4613731c9776e713b075621d23f8ae
   3. c13e1fd2daeaf0538eac92a30c3d673e8c9ad5c2
   4. 07e3a8fe9f1e8237fe18e56bbb7907a130e27803

Commit:
ENGCOM-1614: [Backport] Fixed typo mistake in function comment #15396

Commit:
ENGCOM-1612: [Backport] Remove non-existing argument #15394

magento-engcom-team committed:
:arrows_clockwise: [EngCom] Public Pull Requests - 2.1-develop

Accepted Public Pull Requests:
 - magento/magento2#15396: [Backport] Fixed typo mistake in function comment (by @dmytro-ch)
 - magento/magento2#15394: [Backport] Remove non-existing argument (by @dmytro-ch)

magento-engcom-team committed:
ENGCOM-1624: [Backport] Add 'const' type support to layout arguments #15392

 - Merge Pull Request magento/magento2#15392 from IgorVitol/magento2:2.1-develop-PR-port-15058
 - Merged commits:
   1. 37235814fafa230472df4da891405cbaa1530c7f

rogyar committed:
Fix to allow use decimals less then 1 in subproducts qty

magento-engcom-team committed:
ENGCOM-1626: [Backport] Allow configuring min and max dates for date picker component #15318

 - Merge Pull Request magento/magento2#15318 from rogyar/magento2:2.1-min-max-dates-datepicker
 - Merged commits:
   1. 02fc8c4a234c950ad780d13db2195bb462fe3d25

magento-engcom-team committed:
ENGCOM-1629: [Backport] Eliminate usage of 'else' statements #15309

 - Merge Pull Request magento/magento2#15309 from rogyar/magento2:2.1-eliminate-else-usage
 - Merged commits:
   1. b13d09a6101e37956ae68f8b106f6e9ee6b5e3e4

VitaliyBoyko committed:
Fixed typo in MagentoUi abstract.js

magento-engcom-team committed:
ENGCOM-1638: [Backport-2.1] Fixed typo in MagentoUi abstract.js #15412

 - Merge Pull Request magento/magento2#15412 from VitaliyBoyko/magento2:Fix-abstract.js-typo-2.1
 - Merged commits:
   1. 47dc21025f5045b0f4f18b9c6ff4e69022bfe259

Commit:
ENGCOM-1638: [Backport-2.1] Fixed typo in MagentoUi abstract.js #15412

Commit:
ENGCOM-1629: [Backport] Eliminate usage of "else" statements #15309

magento-engcom-team committed:
ENGCOM-1639: [Backport] Fix to allow use decimals less then 1 in subproducts qty #15407

 - Merge Pull Request magento/magento2#15407 from rogyar/magento2:2.1-js-validation-fix
 - Merged commits:
   1. e8087c5569b32d193801f63fa2c9fd02f10c6a29

AnshuMishra17 committed:
typo correction

magento-engcom-team committed:
:arrows_clockwise: [EngCom] Public Pull Requests - 2.1-develop

Accepted Public Pull Requests:
 - magento/magento2#15412: [Backport-2.1] Fixed typo in MagentoUi abstract.js (by @VitaliyBoyko)
 - magento/magento2#15309: [Backport] Eliminate usage of "else" statements (by @rogyar)

rogyar committed:
Removed redundant else statement

yogeshks committed:
Merge pull request #1 from Yogeshks/2.1-develop-PR-port-15333

[Backport] Added language translation for message string

magento-engcom-team committed:
ENGCOM-1649: [Backport] Added language translation for message string #15444

 - Merge Pull Request magento/magento2#15444 from Yogeshks/magento2:2.1-develop
 - Merged commits:
   1. d2a3b1c575fd4dad54e4df99d4ea07303f8f323a
   2. 9d01d76afb05699c596063a1526341e3be05e17c
   3. 8efc333d8f595ece59cb12f00db204a652c1b101
   4. 002a7c09b18db57027fecebceebeb41743f85c88
   5. 0613e5800c1c3731e36cf4c4fed79d7d92c7d1f0

magento-engcom-team committed:
ENGCOM-1650: [Backport] Removed redundant else statement #15436

 - Merge Pull Request magento/magento2#15436 from rogyar/magento2:2.1-configurable-redundant-else
 - Merged commits:
   1. ec092def9dce169493d1f70741d32d66747a3df5

Commit:
ENGCOM-1639: [Backport] Fix to allow use decimals less then 1 in subproducts qty #15407

Commit:
ENGCOM-1626: [Backport] Allow configuring min and max dates for date picker component #15318

Commit:
ENGCOM-1624: [Backport] Add 'const' type support to layout arguments #15392

magento-engcom-team committed:
:arrows_clockwise: [EngCom] Public Pull Requests - 2.1-develop

Accepted Public Pull Requests:
 - magento/magento2#15407: [Backport] Fix to allow use decimals less then 1 in subproducts qty (by @rogyar)
 - magento/magento2#15318: [Backport] Allow configuring min and max dates for date picker component (by @rogyar)
 - magento/magento2#15392: [Backport] Add 'const' type support to layout arguments (by @IgorVitol)


Fixed GitHub Issues:
 - magento/magento2#14692: 'validate-grouped-qty' validation is meaningless (reported by @likemusic) has been fixed in magento/magento2#15407 by @rogyar in 2.1-develop branch
   Related commits:
     1. e8087c5569b32d193801f63fa2c9fd02f10c6a29

magento-engcom-team committed:
ENGCOM-1660: [Backport] typo correction #15465

 - Merge Pull Request magento/magento2#15465 from mzeis/magento2:2.1-develop-PR-port-15431
 - Merged commits:
   1. fe61969894a1515a0e5088411112befe5ca71fe0

Commit:
ENGCOM-1650: [Backport] Removed redundant else statement #15436

Commit:
ENGCOM-1649: [Backport] Added language translation for message string #15444

magento-engcom-team committed:
:arrows_clockwise: [EngCom] Public Pull Requests - 2.1-develop

Accepted Public Pull Requests:
 - magento/magento2#15436: [Backport] Removed redundant else statement (by @rogyar)
 - magento/magento2#15444: [Backport] Added language translation for message string (by @Yogeshks)

Commit:
ENGCOM-1660: [Backport] typo correction #15465

magento-engcom-team committed:
:arrows_clockwise: [EngCom] Public Pull Requests - 2.1-develop

Accepted Public Pull Requests:
 - magento/magento2#15465: [Backport] typo correction (by @mzeis)

magento-engcom-team committed:
ENGCOM-1685: [backport 2.1] Category\Collection::joinUrlRewrite should use the store set on the collection #13756

 - Merge Pull Request magento/magento2#13756 from alepane21/magento2:bugfix/issue-13704-backport-2.1
 - Merged commits:
   1. 2239f3d21bb1ce9cee807a28cc4a2861b0361ab7
   2. 6f556ff03d8161686e4a49217342756ebd77d6f1
   3. 915be6ea120a8e46922250459f63d077ae8675b0

Commit:
ENGCOM-1685: [backport 2.1] Category\Collection::joinUrlRewrite should use the store set on the collection #13756

rogyar committed:
Add resetting of triggerRecollection flag

VitaliyBoyko committed:
Fix typos in Multishipping and User module

magento-engcom-team committed:
ENGCOM-1693: [Backport] Fix unnecessary recalculation of product list pricing #15445

 - Merge Pull Request magento/magento2#15445 from JeroenVanLeusden/magento2:2.1-develop-PR-port-15089
 - Merged commits:
   1. 3d05e6b5361d28877ac97e7dfd264b8b5ccf6482

magento-engcom-team committed:
ENGCOM-1696: [Backport 2.1]Fix typos in Multishipping and User module #15548

 - Merge Pull Request magento/magento2#15548 from VitaliyBoyko/magento2:Fix-typos-in-Multishipping-and-User-module2-1
 - Merged commits:
   1. 485fd4858240cdcaf834e57a3e3cc081a836f08e

rogyar committed:
Correct function return statement

magento-engcom-team committed:
ENGCOM-1699: [Backlog] Add resetting of triggerRecollection flag #15522

 - Merge Pull Request magento/magento2#15522 from rogyar/magento2:2.1-unset-quote-recollect
 - Merged commits:
   1. bddb976d9a13da3a2c40301e31791469a0e62669

magento-engcom-team committed:
ENGCOM-1708: [Backport] Correct functions return statement #15563

 - Merge Pull Request magento/magento2#15563 from rogyar/magento2:2.1-adjust-return-statements
 - Merged commits:
   1. 7c65f0a9a5bcebe15caaf5d594e5852888da4776

Commit:
ENGCOM-1696: [Backport 2.1]Fix typos in Multishipping and User module #15548

Commit:
ENGCOM-1685: [backport 2.1] Category\Collection::joinUrlRewrite should use the store set on the collection #13756

dmanners commented:
Hi @adam-paterson feel free to update this to match the other PR or if you would prefer you can use our backport tool. https://github.com/magento/magento2/wiki/Simple-way-to-port-your-solutions-automatically

VladimirZaets committed:
[backport 2.1] Category\Collection::joinUrlRewrite should use the store set on the collection #13756
- fix integration test

VladimirZaets committed:
Merge pull request #2616 from magento-engcom/2.1-develop-prs

[EngCom] Public Pull Requests - 2.1-develop

dmytro-ch committed:
Fix incorrect type hinting in PHPDocs

magento-engcom-team committed:
ENGCOM-1727: Fix incorrect type hinting in PHPDocs #15619

 - Merge Pull Request magento/magento2#15619 from dmytro-ch/magento2:fix/2.1-phpdocs-incorrect-type-hinting
 - Merged commits:
   1. ab64a2c39358c6f37d27206a16272cc176a33abe

Commit:
ENGCOM-1727: Fix incorrect type hinting in PHPDocs #15619

Commit:
ENGCOM-1708: [Backport] Correct functions return statement #15563

Commit:
ENGCOM-1699: [Backlog] Add resetting of triggerRecollection flag #15522

Commit:
ENGCOM-1693: [Backport] Fix unnecessary recalculation of product list pricing #15445

ishakhsuvarov committed:
:arrows_clockwise: [EngCom] Public Pull Requests - 2.1-develop

Accepted Public Pull Requests:
 - magento/magento2#15619: Fix incorrect type hinting in PHPDocs (by @dmytro-ch)
 - magento/magento2#15563: [Backport] Correct functions return statement (by @rogyar)
 - magento/magento2#15522: [Backlog] Add resetting of triggerRecollection flag (by @rogyar)
 - magento/magento2#15445: [Backport] Fix unnecessary recalculation of product list pricing (by @JeroenVanLeusden)


Fixed GitHub Issues:
 - magento/magento2#13992: Incorrect phpdoc should be Shipment\Item not Invoice\Item (reported by @ThisIsRuddy) has been fixed in magento/magento2#15619 by @dmytro-ch in 2.1-develop branch
   Related commits:
     1. ab64a2c39358c6f37d27206a16272cc176a33abe

 - magento/magento2#9580: Quote Attribute trigger_recollect causes a timeout (reported by @bh-ref) has been fixed in magento/magento2#15522 by @rogyar in 2.1-develop branch
   Related commits:
     1. bddb976d9a13da3a2c40301e31791469a0e62669

 - magento/magento2#14941: Unnecessary recalculation of product list pricing causes huge slowdowns (reported by @paales) has been fixed in magento/magento2#15445 by @JeroenVanLeusden in 2.1-develop branch
   Related commits:
     1. 3d05e6b5361d28877ac97e7dfd264b8b5ccf6482

magento-cicd2 commented:
[![CLA assistant check](https://cla.dev.magento.com/pull/badge/not_signed)](https://cla.dev.magento.com/magento-engcom/import-export-improvements?pullRequest=105) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla.dev.magento.com/magento-engcom/import-export-improvements?pullRequest=105) before we can accept your contribution.<br/>**0** out of **6** committers have signed the CLA.<br/><br/>:x: slavvka<br/>:x: rogyar<br/>:x: sidolov<br/>:x: magento-engcom-team<br/>:x: magento-team<br/>:x: ferrazzuk

dmanners commented:
Hi @adam-paterson I am going to close this PR since it seems to be all messed up with commits etc and also it seems to have been fixed on 2.2-develop by the core team. See https://github.com/magento/magento2/commit/ec8830fcd9b86ce75aa2a3cc685fa83316dcc971 for the core fix.
